README.md

```diff
@@ -34,6 +34,37 @@ Once you're logged in,
 4. Activate/Install other plugins you will need for your development (i.e `custom posts`, `advance custom fields`, `ACF to REST API`)
 5. All done! You can now add content, use and customize plugins.
 
+
+### Child Theme
+
+To create a child theme with the hurumap as the parent theme;
+
+ - Create a folder in the  `wp-content/themes` folder  with the desired name eg (hurumap-child)
```

Comment:
```suggestion
 - Create a folder in the  `wp-content/themes` folder  with the desired name e.g. (hurumap-outbreak-africa)
```
outbreak.AFRICA is a more established name at the moment (has it's own domain even) so lets use that as an example.